ui/src/data_explorer/components/WriteDataBody.tsx

```diff
@@ -7,6 +7,7 @@ import React, {
   ReactElement,
 } from 'react'
 import WriteDataFooter from 'src/data_explorer/components/WriteDataFooter'
+import {WriteDataMode} from 'src/shared/constants'
```

Comment:
this could also be an enum of strings instead of an object of constants, for stricter typing